pkg/cli/sql.go

```diff
@@ -414,6 +414,13 @@ func preparePrompts(dbURL string) (promptPrefix, fullPrompt, continuePrompt stri
 		username := ""
 		if parsedURL.User != nil {
 			username = parsedURL.User.Username()
+		} else {
+			u, err := user.Current()
+			if err == nil {
+				username = u.Username
+			} else {
+				username, _ = envutil.EnvString("COCKROACH_USER", 2)
```

Comment:
I think it would be wrong to do this because we would not be using `COCKROACH_USER` to connect to the server (since we prefer url parameters over connection flags/environment variables) so there would be a disconnect between the user shown in the prompt and the actual session user. I would just remove this.